### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by hazg** [trbrc/svelte-inspect#7]
Duplicate slot name "slot" in <Echo>

in https://svelte.dev/repl/eb3b3ae5639544d78d7363e126b29896?version=3.35.0

**Comment by brunetton** [comment]
@trbrc any chances that it could be updated ?

Thanks !

**Comment by frantic0** [comment]
hi @trbrc 

on a quick check of the svelte-inspect source, DateFormatters and SymbolFormatter use `<Echo>` which has duplicate slot names 

```
  <Echo>
    <span slot=slot class=affix>{prefix}</span>
    <span slot=slot class=key>{key}</span>
    <span slot=slot class=affix>)</span>
  </Echo>
```
It seems we can't have duplicate slot names in svelte. If you try to change in the svelte tutorial `name-slots` https://svelte.dev/tutorial/named-slots, the compiler will complain about duplicate slot names.

I didn't try to understand the overall component logic, but I am wondering whether this isn't trivial to fix?

**Comment by alfrednerstu** [comment]
I have the same issue

`Duplicate slot name "slot" in <Echo>`

**Comment by trbrc** [comment]
Thanks for your patience. It's fixed in 0.1.3!
